src/sage/interfaces/maxima_abstract.py

```diff
@@ -189,9 +189,7 @@ def help(self, s):
 
         - ``s`` -- string
 
-        OUTPUT:
-
-        Maxima's help for ``s``
+        OUTPUT: maxima's help for ``s``
```

Comment:
I think this is a case where we should preserve capitalization. And I think calling the output a `Maxima function` is more helpful than trying to be more technically precise.

```suggestion
        OUTPUT: Maxima's help for ``s``
```